server/config.ts

```diff
@@ -77,9 +77,11 @@ export default {
     },
     interventionsService: {
       url: get('INTERVENTIONS_SERVICE_URL', 'http://localhost:9092', requiredInProduction),
+      // the interventions-service <-> community-api worst case timeout is 20s, this is intentionally just slightly higher
+      // fixme: this is too long
       timeout: {
-        response: 10000,
-        deadline: 10000,
+        response: 22000,
+        deadline: 22000,
```

Comment:
We can go up to `30s` I think as long as we put the end-to-end transaction on a KPI and set a target to bring it down